xWorkflows/xWorkflow_2/flask_backend/app.py

```diff
@@ -8,28 +8,41 @@
 
 # Connect to MongoDB
 client = MongoClient("mongodb://admin:password@mongo-db-0.mongo-db.default.svc.cluster.local:27017")
```

Comment:
Pass username and password using env

Reply:
There is a separate ticket already created for this, in this ticket I worked on error handling. The PR for this code was already merged so we can address this in the env and secrets ticket.